migration_steps/shared/validation_mapping.json

```diff
@@ -1373,8 +1373,9 @@
             ],
             "exception_table_join": "LEFT JOIN {casrec_schema}.exceptions_deputy_addresses exc_table ON exc_table.caserecnumber = {casrec_schema}.order.\"Case\"",
             "where_clauses": [
-                "COALESCE({casrec_schema}.deplink.\"Main Addr\", 'NO MAIN ADDRESS') IN ('1', 'NO MAIN ADDRESS')",
-                "{casrec_schema}.deputy.\"Dep Type\" IN ('20','21','22','24','25','26','27','28','29','63','71')"
+                "((COALESCE(NULLIF({casrec_schema}.deplink.\"Main Addr\", ''), 'NO MAIN ADDRESS') IN ('1', 'NO MAIN ADDRESS') AND {casrec_schema}.deputy.\"Dep Type\" IN ('20','21','22','24','25','26','27','28','29','63','71')) OR {casrec_schema}.deputy.\"Dep Type\" NOT IN ('20','21','22','24','25','26','27','28','29','63','71'))",
```

Comment:
I think my logic was wrong here - `IN ('1', 'NO MAIN ADDRESS')` - because it will basically get all Main and non-Main addresses. Assuming that a deputy can have multiple addresses, we want to select 'NO MAIN ADDRESS' only when there is no Main address for that deputy. Which I think is doable with a Lateral join or a nested select in the where clause

Reply:
ah, yes, I think you're right; it should just be "Main Addr" = '1' (I think)

Reply:
Probably best to set "Main Addr" = '1'.